conans/client/tools/system_pm.py

```diff
@@ -126,6 +126,64 @@ def install(self, packages, update=True, force=False, arch_names=None):
             self.update()
         self._install_any(packages)
 
+    def install_packages(self, packages, update=True, force=False, arch_names=None):
+        """ Get the system package tool install command and install all packages and/or variants.
+            Inputs:
+            "pkg-variant1"  # (1) Install only one package
+            ["pkg-variant1", "otherpkg", "thirdpkg"] # (2) All must be installed
+            [("pkg-variant1", "pkg-variant2"), "otherpkg", "thirdpkg"] # (3) Install only one variant
+                                                                             and all other packages
+            "pkg1 pkg2", "pkg3 pkg4" # (4) Invalid
+            ["pkg1 pkg2", "pkg3 pkg4"] # (5) Invalid
+
+        :param packages: Supports multiple formats (string,list,tuple). Lists and tuples into a list
+        are considered variants and is processed just like self.install(). A list of string is
+        considered a list of packages to be installed (only not installed yet).
+        :param update: Run update command before to install
+        :param force: Force installing all packages, including all installed.
+        :param arch_names: Package suffix/prefix name used by installer tool e.g. {"x86_64": "amd64"}
+        :return: None
+        """
+        packages = [packages] if isinstance(packages, str) else list(packages)
```

Comment:
Unicode (py2) not expected here, but just in case.

```suggestion
        packages = [packages] if isinstance(packages, six.string_types) else list(packages)
```

Reply:
Done.